assets/stylesheets/components/_toast.scss

```diff
@@ -0,0 +1,44 @@
+#toast-container {
```

Comment:
@tcmacdonald: Correct. Imposed by the library.

The class on the container is specifically to be able to position it as needed, so we're a bit stuck with the `id` in this case.

I don't love this approach because an update to the library could break our styles, but I don't have a better way to get around it.